src/Attestation/Attestation/Commands/GetAzureAttestation.cs

```diff
@@ -65,19 +68,62 @@ public class GetAzureAttestation : AttestationManagementCmdletBase
         [ResourceGroupCompleter]
         [ValidateNotNullOrEmpty()]
         public string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Location of the default attestation.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            Position = 0,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = DefaultProviderParameterSet,
+            HelpMessage = "Specifies the Location of the default attestation provider.")]
+        [ValidateNotNullOrEmpty]
+        public string Location { get; set; }
+
+        /// <summary>
+        /// Flag for the default attestation.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = DefaultProviderParameterSet,
+            HelpMessage = "Specifies this is the request to a default attestation provider.")]
+        [ValidateNotNullOrEmpty]
+        public SwitchParameter DefaultProvider { get; set; }
         #endregion
 
         public override void ExecuteCmdlet()
         {
-            if (ResourceId != null)
+            PSAttestation attestation;
+
+            switch (ParameterSetName)
             {
-                var resourceIdentifier = new ResourceIdentifier(ResourceId);
-                Name = resourceIdentifier.ResourceName;
-                ResourceGroupName = resourceIdentifier.ResourceGroupName;
-            }
+                case DefaultProviderParameterSet:
+                    if (string.IsNullOrEmpty(Location))
+                    {
+                        this.WriteObject(AttestationClient.ListDefaultAttestation());
```

Comment:
use
this.WriteObject(AttestationClient.ListDefaultAttestation(), true);
for collection

Reply:
Fixed.